dissect/target/plugins/os/unix/bsd/citrix/_os.py

```diff
@@ -96,16 +116,17 @@ def ips(self) -> list[str]:
     @export(record=UnixUserRecord)
     def users(self) -> Iterator[UnixUserRecord]:
         nstmp_users = set()
-        nstmp_path = "/var/nstmp/"
+        seen = set()
```

Comment:
Could you "uniq" on `(name, home, shell)`? Shell will mostly be None for the Citrix specific users, but it's good practice.

Reply:
Done